src/components/LaunchFilter/types/LaunchesAction.ts

```diff
@@ -0,0 +1,16 @@
+import { Launch } from '../../../shared/api/types/Launch';
+import { LaunchFilters } from '../constants/filerConstants';
+
+export interface FilterActionPayload {
+  filterValue: LaunchFilters;
+  error: string;
```

Comment:
1. why an error is required?
2. don't split types for action and payload

Reply:
2. United.